src/viking.js

```diff
@@ -1,11 +1,125 @@
+// Link to pull: https://github.com/ironhack-labs/lab-javascript-vikings/pull/3889
+
+
 // Soldier
-class Soldier {}
+class Soldier {
+
+    constructor(health, strength){
+        this.health = health
+        this. strength = strength
+    }
+    attack(){
+        return this.strength
+
+    }
+    receiveDamage(damage) {
+       
+        this.health -= damage;
+      }
+
+
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor(name, health,strength){
+        super(health,strength)
+        this.name = name
+    }
+    receiveDamage(damage) {
+        this.damage=damage
+        this.health -= damage;
+        if(this.health>0){
+            return `${this.name} has received ${damage} points of damage`
+        }
+        else{
+            return `${this.name} has died in act of combat`
+        }
+       
+
+        }
+        battleCry(){
+            return "Odin Owns You All!"
+        }
+        
+      }
+
+
+
 
 // Saxon
-class Saxon {}
+class Saxon extends Soldier{
+    receiveDamage(damage){
+        this.health -= damage
+        if(this.health>0){
+        return `A Saxon has received ${damage} points of damage`}
+        else{
+            return  "A Saxon has died in combat"
+        }
+
+    }
+}
 
 // War
-class War {}
+class War {
+   constructor(){
+    this.vikingArmy=[]
+    this.saxonArmy=[]
+
+   }
+   addViking(vickingWarriors){
+    this.vikingArmy.push(vickingWarriors)
+
+   }
+   addSaxon(SaxonWarriors){
+    this.saxonArmy.push(SaxonWarriors)
```

Comment:
In those methods you're using a different indentation from the previous one
```suggestion
      this.saxonArmy.push(SaxonWarriors)
```